themes/bootstrap3/templates/RecordTab/holdingsils.phtml

```diff
@@ -55,7 +55,7 @@
   <h3><?=$this->transEsc("Internet")?></h3>
   <?php if (!empty($urls)): ?>
     <?php foreach ($urls as $current): ?>
-      <a href="<?=$this->escapeHtmlAttr($this->proxyUrl($current['url']))?>"><?=$this->escapeHtml($current['desc'])?></a><br/>
+      <a href="<?=$this->escapeHtmlAttr($this->proxyUrl($current['url']))?>" property="sameAs"><?=$this->escapeHtml($current['desc'])?></a><br/>
```

Comment:
This might be too strong of a statement. We don't really know what the URLs associated with a record are; they might be a representation, or they might be supplemental information. I don't think we can guarantee that they are always "sameAs."

Reply:
Agree, i will remove it